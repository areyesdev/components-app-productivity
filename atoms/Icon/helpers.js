const ICONS_ASSETS_PATH = '/icons',
  iconMap = {
    arrowRight: 'arrow-right',
    angleDown: 'angle-down',
    iconPlus: 'icon-plus',
  },
  iconSize = {
    sm: 15,
    md: 26,
    lg: 40,
    xl: 40,
  }

export const mapType = (type) => `${ICONS_ASSETS_PATH}/${iconMap[type]}.svg`
export const mapSize = (size) => iconSize[size]
