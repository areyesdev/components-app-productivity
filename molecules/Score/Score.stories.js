import { Score, styles } from '.'

import { getTemplate } from '../../helpers/storybook'

const Template = getTemplate(Score, styles)

export default {
  title: 'Molecules/Score',
  component: Score,
  args: {},
}

export const Default = Template.bind({})
