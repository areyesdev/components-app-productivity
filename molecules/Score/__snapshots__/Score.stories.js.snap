// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Molecules/Score Default 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="score"
  >
    <div
      className="face"
      onClick={[Function]}
    >
      <SvgrURL />
      <div
        className="spacer vertical"
        style={
          Object {
            "height": 5,
          }
        }
      />
      <p
        className="paragraph color-base size-md weight-normal"
      >
        Frustrado
      </p>
    </div>
    <div
      className="spacer horizontal"
      style={
        Object {
          "height": "100%",
          "maxHeight": "auto",
          "width": 40,
        }
      }
    />
    <div
      className="face"
      onClick={[Function]}
    >
      <SvgrURL />
      <div
        className="spacer vertical"
        style={
          Object {
            "height": 5,
          }
        }
      />
      <p
        className="paragraph color-base size-md weight-normal"
      >
        Normal
      </p>
    </div>
    <div
      className="spacer horizontal"
      style={
        Object {
          "height": "100%",
          "maxHeight": "auto",
          "width": 40,
        }
      }
    />
    <div
      className="face"
      onClick={[Function]}
    >
      <SvgrURL />
      <div
        className="spacer vertical"
        style={
          Object {
            "height": 5,
          }
        }
      />
      <p
        className="paragraph color-base size-md weight-normal"
      >
        Muy motivado
      </p>
    </div>
  </div>
</div>
`;
